xcube/core/gridmapping/base.py

```diff
@@ -170,15 +170,16 @@ def derive(
             if other.tile_size != tile_size:
                 other._tile_size = tile_width, tile_height
                 with self._lock:
-                    if other._xy_coords is not None:
-                        other._xy_coords = other._xy_coords.chunk(
-                            {
-                                dim: size
-                                for (dim, size) in zip(
-                                    other._xy_coords.dims, other.xy_coords_chunks
-                                )
-                            }
-                        )
+                    if other._xy_coords is None:
+                        _ = other.xy_coords
```

Comment:
From code inspection and context, I realize that we're forcing initialization of `other._xy_coords` here, but a brief explanatory comment would be welcome :)

Reply:
done